rotkehlchen/api/v1/schemas.py

```diff
@@ -977,15 +974,13 @@ class ExchangesResourceRemoveSchema(Schema):
     location = LocationField(limit_to=SUPPORTED_EXCHANGES, required=True)
 
 
-class ExchangeBalanceQuerySchema(Schema):
+class ExchangeBalanceQuerySchema(AsyncQueryArgumentSchema):
     location = LocationField(limit_to=SUPPORTED_EXCHANGES, load_default=None)
-    async_query = fields.Boolean(load_default=False)
     ignore_cache = fields.Boolean(load_default=False)
 
 
-class BlockchainBalanceQuerySchema(Schema):
+class BlockchainBalanceQuerySchema(AsyncQueryArgumentSchema):
```

Comment:
same as above. This is used here: 

https://github.com/rotki/rotki/blob/98f49cd3ed26c641fec03b78eff9fe1872385fbf/rotkehlchen/api/v1/resources.py#L584

Also wonder why tests did not catch this. Find out why.